For whatever insane reason, the xvtiffwr.c module writes all tiffs out with
resolution tags of 1200dpi.  This leaps up and bites you on the ass when
you load the resultant tiff files into a program (such as Photoshop) that
actually pays attention to said tags.

This patch simply writes all tiff files with a res of 72dpi, a far more 
reasonable value.

--jhb, 3/7/2000


*** xvtiffwr.c.orig	Tue Jan  3 16:28:13 1995
--- xvtiffwr.c	Tue Mar  7 12:19:13 2000
***************
*** 84,91 ****
    TIFFSetField(tif, TIFFTAG_ROWSPERSTRIP, h);
  
    TIFFSetField(tif, TIFFTAG_RESOLUTIONUNIT, (int)2);
!   TIFFSetField(tif, TIFFTAG_XRESOLUTION, (float)1200.0);
!   TIFFSetField(tif, TIFFTAG_YRESOLUTION, (float)1200.0);
  
  
    /* write the image data */
--- 84,91 ----
    TIFFSetField(tif, TIFFTAG_ROWSPERSTRIP, h);
  
    TIFFSetField(tif, TIFFTAG_RESOLUTIONUNIT, (int)2);
!   TIFFSetField(tif, TIFFTAG_XRESOLUTION, (float) 72.0);
!   TIFFSetField(tif, TIFFTAG_YRESOLUTION, (float) 72.0);
  
  
    /* write the image data */
